shared/chat/inbox/container.js

```diff
@@ -101,11 +101,13 @@ const getRowsAndMetadata = createSelector(
   [getVisibleSmallIDs, _smallTeamsPassThrough, getBigRowItems],
   (smallIDs, smallTeamsExpanded, bigRows) => {
     const smallTeamsRowsToHideCount = Math.max(0, smallIDs.length - smallTeamsCollapsedMaxShown)
-    const smallIDsToShow = smallTeamsExpanded ? smallIDs : smallIDs.slice(0, smallTeamsCollapsedMaxShown)
+    const showSmallTeamsExpandDivider = !!(bigRows.length && smallTeamsRowsToHideCount && !smallTeamsExpanded)
```

Comment:
No more collapsing ad-hoc conversations?

Reply:
you can (i think). we only allow collapsing if
1. you have big teams
2. you have at least 5 small teams
3. its expanded

i tested this a little and it seems correct
if you have no big teams, we see all small rows
if you have 3 small rows we don't see the divider and we see all 3
if you have 6 small rows we show 1 hidden and the divider and the divider works

Reply:
hmm, when I open my inbox and click the `+20 more`, it expands out my conversations and below them, shows my big teams + channels as expected. But, no collapse button to be found, is that expected?

Reply:
nope, fixing now tnx!